renovate.json

```diff
@@ -1,7 +1,8 @@
 {
   "$schema": "https://docs.renovatebot.com/renovate-schema.json",
   "extends": [
-    "config:base"
+    "config:base",
+    "helpers:pinGitHubActionDigests"
```

Comment:
is this needed? I thought renovatebot would detect our config and update? (assuming we already have all action pinned by hash)

Reply:
No, it doesn't default, AFAIK. We would have to opt in.

Reply:
The [PR](https://github.com/slsa-framework/slsa-github-generator/pull/401) that enabled renovatebot understood the pin that were already used, e.g., `Upgrade actions/checkout to 2541b1294d2704b0964813337f33b291d3f8596b`

IIIC, the additional option is used to migrate version pins to hash pins, or have both tag/hash (dockerfiles), e.g., https://docs.renovatebot.com/configuration-options/#pindigest

Reply:
OK. Updated to include the pindigests.